hw/ip/gpio/data/gpio.hjson

```diff
@@ -110,7 +110,9 @@
       act:     "rcv",
       package: "",
       desc:    '''
-               This signal is pulsed high by the power manager after reset in order to sample the HW straps.
+               The strap enable signal is used to make the snapshot of the gpio_in.
```

Comment:
English nit:
- It's a signal to make a snapshot, rather than a mechanism by which to make the snapshot.
- gpio_in probably doesn't need a "the". (But I'd weaken the text slightly to...)

Maybe better: "The strap enable signal tells gpio to take a snapshot of the input pins" ?

Reply:
I think this suggestion is just adding "input" to my text? Either is fine, but something needs to change because the text currently in the PR says that the strap enable signal is the mechanism by which something makes a snapshot, rather than an instruction that tells something to make a snapshot.